core/Migrations/Version28000Date20231126110901.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * @copyright 2023 Maxence Lange <maxence@artificial-owl.com>
+ *
+ * @author Maxence Lange <maxence@artificial-owl.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OC\Core\Migrations;
+
+use Closure;
+use OCP\DB\ISchemaWrapper;
+use OCP\DB\Types;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+/**
+ * Create new fields for type and lazy loading in appconfig for the new IAppConfig API.
+ */
+class Version28000Date20231126110901 extends SimpleMigrationStep {
```

Comment:
```suggestion
class Version29000Date20240123... extends SimpleMigrationStep {
```
Please generate a proper new migration that executes **after** the already existing one, so there is a proper stream line of migrations and an easy traceable DB state of instanced running dailies or git.

Reply:
Okay I see only now that you changed the version number in the backport. That should not have happened.

Anyway, we should still have the new and fixing migration with a 2401... Timestamp at the end of the chain and the 28...2311 migrations both unchanged from previous PRs.